### PR TITLE
Template suggestion for HW1

### PR DESCRIPTION
**Pull request by hannelita**
Questions:

* If I remember correctly, that was the project structure we used for most of the projects. I just pulled out the `test` directory  to the submission root
* The idea will be that students use the structure in the `submission_template`; we suggest they should clone the repo and rename the directory before submitting it on mycourses 
* `results.tsv` and `dataset.tsv` should be on inside submission_template
* How further should we test here? I'm trying to guarantee some basic structure for the files
* I'm also using some specific libs - any plan to guarantee libs are properly installed? 
 
 

**I'll write up all these instructions later after you approve this plan**

### CODE REVIEW

**Comment by hannelita** [comment]
@hmsaleem ^

**Comment by druths** [comment]
Nice!  Looks good to me.

In terms of checking library versions, you could import them and actually check the version... e.g.,

import pandas
assert pandas.__version__ == '2.0.1'

**Comment by hannelita** [comment]
> Nice! Looks good to me.
> 
> In terms of checking library versions, you could import them and actually check the version... e.g.,
> 
> ```
> import pandas
> assert pandas.__version__ == '2.0.1'
> ```

What about adding a `requirements.txt` ?

File: hw1/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+**Important instructions** 
+Please read these instructions before submitting your assignment. 
+If you do not follow the template specified here, **we will not able to grade your assignment**! 
+To avoid pain and discomfort, follow the steps below carefully:
+
+## Getting this code
+
+This code contains the templates for every HW assignment in this course.
+Download the code from Github.
+Or, if you are familiar with Git, clone this repository.
```
[the remark this answers is not in the document]
Comment (hannelita) [reply hw1/README.md:12]: Cool, I'll move it to the `Other tips` section

**Pull request merged**